controllers/admin/AdminImagesController.php

```diff
@@ -892,6 +892,22 @@ protected function regenerateNewImage($entityType, $idEntity)
                                     'webp'
                                 );
                             }
+
+                            if (ImageManager::retinaSupport()) {
+                                if (!ImageManager::resize($existingImage, $process[$entityType].$imageObj->getExistingImgPath().'-'.stripslashes($imageType['name']).'2x.jpg', (int) $imageType['width'] * 2, (int) $imageType['height'] * 2)) {
```

Comment:
Looks like the occurrances of this method are splitted into multiple lines, can we do it for this line as well? It would help with readability.